src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -90,8 +94,8 @@ interface ItemListLayoutUserSettings {
 @observer
 export class ItemListLayout extends React.Component<ItemListLayoutProps> {
   static defaultProps = defaultProps as object;
-
   @observable isUnmounting = false;
+  @observable columnswidth = new Array<number>(this.props.renderTableHeader.length);
```

Comment:
This being an array is brittle, especially since in the future we want to be able to support moving these columns around. This should be a map between the column IDs (`className` should work I think) and widths.